### PR TITLE
[luci/pass] Reduce duplicates of ForwardTransposeOpPass

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit reduces duplicates of ForwardTransposeOpPass.

Related : #9947 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp [hunk 1]
```diff
@@ -308,6 +335,14 @@ TEST_F(ForwardTransposeToAddGraphTest, forward_transpose_add_NEG)
...
 }
 
+TEST_F(ForwardTransposeToAddGraphNegTest, forward_transpose_add_non_const_NEG)
+{
+  _graph.init({1, 64, 51, 1}, {0, 3, 2, 1});
+
+  luci::ForwardTransposeOpPass pass;
+  EXPECT_FALSE(pass.run(_graph.g()));
+}
+
```
Comment (jinevening) [inline compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:345]: Why is this test added in this PR?

Comment (mhs4670go) [reply compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:345]: Well, for the case where binary op has no const operaters. Is this case going to be covered later?

Comment (jinevening) [reply compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:345]: Adding a new test seems not related to reducing duplicate codes.

Is this for increasing the portion of negative test?

Comment (mhs4670go) [reply compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:345]: Ah, sort of. The test is included to prevent minor bugs that may occur during refactoring.

Comment (jinevening) [reply compiler/luci/pass/src/ForwardTransposeOpPass.test.cpp:345]: I see.

**Review by seanshpark — APPROVED** [review]
